### ISSUE TITLE
About searilization

### ISSUE THREAD

**Issue by SpawnProd** [jacqueskang/IpcServiceFramework#117]
As we know, newtonjson become standard for asp.net, but since .net core 3 we get System.Text.Json, how about to remove newtonjson dependency?

Some features (like JsonConstructor for example) will be supported only in 5.0, but we can do all stuff and without it.

System.Text.Json is part of .net core for now and even for asp.net core newtonjson is just option.

As solution we can separate all serialization stuff and realise system.text.json, newtonjson, and many other alternatives, if somebody need them.

**Comment by jacqueskang** [comment]
Indeed, I was thinking about migrating to System.Text.Json. I have a little bit doubt about the polymorphic serialization but it worth a try.

**Comment by SpawnProd** [comment]
> I have a little bit doubt about the polymorphic serialization but it worth a try.

The only way - custom JsonConverter. In simple (but with huge amount of objects) examples System.Json.Text is faster.

**Comment by jacqueskang** [comment]
I'm moving this to next milestone due of lack of time. Sorry
